### PR TITLE
WE-738 Fix Dropdown and replace MUI with EDS

### PR DESCRIPTION
**Pull request by KindCoder-no**
## Fixes
This pull request fixes WE-738

## Description
Fixed Dropdown on tubular and log properties modal and replaced MUI components with EDS

## Type of change
* Enhancement of existing functionality

## Impacted Areas in Application

* Frontend

## Checklist:

*Communication*
* [X] PR is related to an issue
* [ ] I have made corresponding changes to the documentation

*Code quality*
* [X] Code follows the style guidelines
* [X] I have self-reviewed my code
* [X] No new warnings are generated

*Test coverage*
* [X] Existing tests pass
* [ ] New code is covered by passing tests

### CODE REVIEW

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx [hunk 1]
```diff
@@ -55,41 +55,32 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
                 id="uid"
                 label="uid"
                 value={editableLogObject.uid}
-                fullWidth
                 disabled={editMode}
                 required
-                error={!validText(editableLogObject.uid)}
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:72]: The equivalent of `error={!validText(editableLogObject.uid)}` in EDS is `variant={validText(editableLogObject.uid) ? undefined : "error"}`

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx [hunk 2]
```diff
@@ -55,41 +55,32 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
...
                 helperText={editableLogObject.uid.length === 0 ? "A wellbore uid must be 1-64 characters" : ""}
-                inputProps={{ minLength: 1, maxLength: 64 }}
-                onChange={(e) => setEditableLogObject({ ...editableLogObject, uid: e.target.value })}
+                onChange={(e: any) => setEditableLogObject({ ...editableLogObject, uid: e.target.value })}
               />
               <TextField
                 id="name"
                 label="name"
                 defaultValue={editableLogObject.name}
-                error={editableLogObject.name.length === 0}
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:81]: Same as line 61

File: Src/WitsmlExplorer.Frontend/components/Modals/TubularPropertiesModal.tsx [hunk 3]
```diff
@@ -55,33 +55,23 @@ const TubularPropertiesModal = (props: TubularPropertiesModalInterface): React.R
           heading={`Edit properties for ${editableTubular.name}`}
           content={
             <>
-              <TextField disabled id="uid" label="uid" defaultValue={editableTubular.uid} fullWidth />
+              <TextField disabled id="uid" label="uid" defaultValue={editableTubular.uid} />
               <TextField
                 id="name"
                 label="name"
                 defaultValue={editableTubular.name}
-                error={editableTubular.name.length === 0}
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Modals/TubularPropertiesModal.tsx:63]: Also here

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx (hunk 1 above)
Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:72]: Fixed

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx (hunk 2 above)
Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:81]: Fixed

File: Src/WitsmlExplorer.Frontend/components/Modals/TubularPropertiesModal.tsx (hunk 3 above)
Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/TubularPropertiesModal.tsx:63]: Fixed

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx [hunk 4]
```diff
@@ -42,7 +42,7 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
   };
 
   const onChangeCurve = async (event: any) => {
-    const indexCurve = event.target.value === IndexCurve.Time ? IndexCurve.Time : IndexCurve.Depth;
+    const indexCurve = event[0] === IndexCurve.Time ? IndexCurve.Time : IndexCurve.Depth;
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx]: Should be `event.selectedItems[0]` not `event[0]`

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx [hunk 5]
```diff
@@ -66,41 +66,34 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
...
+              />
+              <TextField disabled id="serviceCompany" label="service company" defaultValue={editableLogObject.serviceCompany} />
+              <TextField disabled id="startIndex" label="start index" defaultValue={editableLogObject.startIndex} />
+              <TextField disabled id="endIndex" label="end index" defaultValue={editableLogObject.endIndex} />
+              <TextField disabled id="wellUid" label="well uid" defaultValue={editableLogObject.wellUid} />
+              <TextField disabled id="wellName" label="well name" defaultValue={editableLogObject.wellName} />
+              <TextField disabled id="wellboreUid" label="wellbore uid" defaultValue={editableLogObject.wellboreUid} />
+              <TextField disabled id="wellboreName" label="wellbore name" defaultValue={editableLogObject.wellboreName} />
+              <Autocomplete
+                label="index curve"
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: Should be `disabled={editMode}`

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx (hunk 4 above)
Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx]: Fixed

File: Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx (hunk 5 above)
Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: On the Textfields?

Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: See last commit

Comment (janburak) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: Only Autocomplete, the same as before the changes.

Comment (janburak) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: image

Comment (KindCoder-no) [reply Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx:91]: Fixed now

**Review by janburak — APPROVED** [review]
- [X] Read issue and PR
- [X] Pulled branch and manually tested
- [X] Verify that PR resolves issue
- [X] Reviewed the code

**Pull request merged**